src/spatial.jl

```diff
@@ -194,6 +194,9 @@ Unitful.ustrip(u::Unitful.Units, b::CubicBoundary) = CubicBoundary(ustrip.(u, b.
 Unitful.ustrip(b::RectangularBoundary) = RectangularBoundary(ustrip.(b.side_lengths))
 Unitful.ustrip(u::Unitful.Units, b::RectangularBoundary) = RectangularBoundary(ustrip.(u, b.side_lengths))
 
+Unitful.unit(b::Union{CubicBoundary,RectangularBoundary}) = unit.(b.side_lengths)
```

Comment:
Are these required for the PR? I'm not sure `unit` should return multiple values, it errors on vectors for example.

Reply:
Not really I just added it for convenience.